examples/xim.rs

```diff
@@ -0,0 +1,27 @@
+extern crate winit;
+extern crate libc;
+extern crate x11_dl;
+
+fn main() {
+    let mut events_loop = winit::EventsLoop::new();
+    unsafe {
+        libc::setlocale(libc::LC_CTYPE, b"\0".as_ptr() as *const _);
```

Comment:
Might be nice to have a comment that passing an empty string will cause the locale to be set from environment variables. Ditto for `XSetLocaleModifiers`.